lib/commands/find.js

```diff
@@ -213,6 +222,41 @@ commands.findUIElementsByXpath = async function (selector, mult, context=null, c
   return res;
 };
 
+commands.findScrollableElOrEls = async function (mult, context) {
+  const scrollTypes = ['UIAScrollView', 'UIATableView', 'UIACollectionView', 'UIAWebView'];
+  let command;
+  let res;
+  let ext = mult ? 's' : '';
+  for (let i = 0; i < scrollTypes.length; i++) {
+    command = `au.getElement${ext}ByType('${scrollTypes[i]}'${context})`;
+    if (mult) {
+      if (!res) {
+        res = [];
```

Comment:
this logic is too complicated, you can use something like `res = [...(res || []), ...elements]` to extend res